projects/uitdatabank/docs/entry-api/events/create-online.md

```diff
@@ -1,13 +1,224 @@
+        "endDate": "2023-01-18T23:00:00+01:00"
+      }
+    ]
+  }
+}
+```
+
+To view the JSON data of your online event afterward, copy the `url` property from the response and open it in your web browser (no authentication required).
+
+## Creating a mixed event
+
+A mixed event is a combination of an online and offline event, for example a concert that is played at a specific venue but also has a livestream at the same time.
+
+To create a mixed event, you must provide both a `location` (as on a regular event) and set the `attendanceMode` to `mixed`.
+
+For example, a concert performed at the Ancienne Belgique and also via livestream:
+
+```http
+POST /events HTTP/1.1
+Host: https://io-test.uitdatabank.be
+Authorization: Bearer YOUR_ACCESS_TOKEN
+Content-Type: application/json
+
+{
+  "mainLanguage": "nl",
+  "name": {
+    "nl": "Voorbeeld concert"
+  },
+  "attendanceMode": "mixed",
+  "location": {
+    "@id": "https://io-test.uitdatabank.be/place/787d7420-c06f-4935-b3c5-5cd5a1276796"
+  },
+  "terms": [
+    {
+      "id": "0.50.4.0.0"
+    }
+  ],
+  "calendarType": "single",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00"
+    }
+  ]
+}
+```
+
+## Adding a URL to attend your online or mixed event
+
+When creating or [updating](./update.md) an online or mixed event, you can optionally include an `onlineUrl` property for visitors to attend your event online. The `onlineUrl` expects a URL string that starts with `http://` or `https://`.
+
+This property is not required because some online or mixed events may require attendees to purchase a ticket first before they gain access to the online event, for example via a link in an email.
+
+For example, to create an online event for a livestream of a concert on YouTube:
+```http
+POST /events HTTP/1.1
+Host: https://io-test.uitdatabank.be
+Authorization: Bearer YOUR_ACCESS_TOKEN
+Content-Type: application/json
+
+{
+  "mainLanguage": "nl",
+  "name": {
+    "nl": "Voorbeeld concert livestream @ YouTube"
+  },
+  "attendanceMode": "online",
+  "onlineUrl": "https://www.youtube.com/watch?v=...",
+  "terms": [
+    {
+      "id": "0.50.4.0.0"
+    }
+  ],
+  "calendarType": "single",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00"
+    }
+  ]
+}
+```
+
+## Updating an event's attendanceMode
+
+You can always change an event's `attendanceMode` after its initial creation by [updating the event](./update.md).
+
+Keep the following restrictions in mind:
+
+* When changing the attendanceMode from `offline` or `mixed` **to** `online`, you must remove the `location` property from the event at the same time (or explicitly set it to the "nil location").
+* When changing the attendanceMode from `online` **to** `offline` or `mixed`, you must add a `location` property with an `@id` that links to the place where the event happens at.
+
```

Comment:
It's not a restriction but maybe some behaviour / backend logic inside UDB that is worth mentioning:

When changing the attendanceMode from `online` or `mixed` to `offline` the `onlineUrl` will automatically be removed by UiTdatabank (if present)

Reply:
Good point, I will add it 👍

Reply:
See [e0d0912](https://github.com/cultuurnet/apidocs/pull/189/commits/e0d0912c03f6c541cf321bb8609f038e55247eea)